Gordon360/Services/HousingService.cs

```diff
@@ -68,6 +75,22 @@ public string[] GetAllApartmentHalls()
             return hallsResult.ToArray();
         }
 
+        /// <summary>
+        /// Gets all names of apartment halls
+        /// </summary>
+        /// <returns> AN array of hall names </returns>
```

Comment:
```suggestion
        /// <returns> An array of hall names </returns>
```
Just a little bit of OCD from me :)